modules/nextflow/src/main/groovy/nextflow/script/ScriptMeta.groovy

```diff
@@ -256,12 +256,12 @@ class ScriptMeta {
         final result = new HashSet(definitions.size() + imports.size())
         // local definitions
         for( def item : definitions.values() ) {
-            if( item instanceof WorkflowDef && item.name != null )
+            if( item instanceof WorkflowDef && item.name )
                 result.add(item.name)
         }
         // processes from imports
         for( def item: imports.values() ) {
-            if( item instanceof WorkflowDef )
+            if( item instanceof WorkflowDef && item.name )
```

Comment:
```suggestion
            if( item instanceof WorkflowDef )
```

This is not needed

Reply:
Not strictly needed but still a good defensive approach.  Why omitting?

Reply:
what are you defending against? it is impossible to import an entry workflow from another module